docs/content/administration/config-cheat-sheet.en-us.md

```diff
@@ -761,6 +761,17 @@ and
 - `SEND_BUFFER_LEN`: **100**: Buffer length of mailing queue. **DEPRECATED** use `LENGTH` in `[queue.mailer]`
 - `SEND_AS_PLAIN_TEXT`: **false**: Send mails only in plain text, without HTML alternative.
 
+## Override Email Headers (`mailer.override_header`)
+
+⚠️ This is empyt by default, use it only if you know what you need it for.
```

Comment:
```suggestion
⚠️ This is empty by default, use it only if you know what you need it for.
```

Reply:
BTW can we use GFM note syntax instead of this emoji? Does our docs renderer support that?

```
> [!NOTE]
> content